rpmlint/checks/BinariesCheck.py

```diff
@@ -31,6 +31,7 @@ class BinariesCheck(AbstractCheck):
     la_file_regex = re.compile(r'\.la$')
     invalid_dir_ref_regex = re.compile(r'/(home|tmp)(\W|$)')
     usr_arch_share_regex = re.compile(r'/share/.*/(?:x86|i.86|x86_64|ppc|ppc64|s390|s390x|ia64|m68k|arm|aarch64|mips|riscv)')
+    python_module_regex = re.compile(r'.*\.\w*(python|pypy)\w*(-\w+){4}\.so')
```

Comment:
Should this include `$` at the end?

Reply:
`re.fullmatch` which behaves the same as '^.*$'

Reply:
ah, thanks